src/tested/vehicle-wheel-animation-test.js

```diff
@@ -28,36 +28,39 @@ AFRAME.registerComponent('wheel', {
       self.main_bone = vehicle.getObjectByName('main_bone');
     });
   },
-  tick: function () {
-    const speed = this.data.speed;
+  tick: function (t,dt) {
+    const speed = this.data.speed/1000; //speed per millisecond
     const wheelDiameter = this.data.wheelDiameter;
+  
 
-    const dist = Math.PI * wheelDiameter;
-    const distx = speed * 0.003;
-    const t = (distx / dist) * (2 * Math.PI);
+    // let distance = speed * t ;
+    // let angle = (distance/(Math.PI*wheelDiameter))*2*Math.PI;
+    // let rateOfRotation = angle/t;
+
+    let rateOfRotation = (2*(speed/wheelDiameter));
 
     if (this.main_bone) {
-      this.main_bone.position.z += distx;
+      this.main_bone.translateY(speed);
```

Comment:
moving the vehicle via bone translation causes rendering issues.
instead, only wheels should be animated with bones and use a-frame / three.js to move the vehicle using animation component

Reply:
the test can use this componet:
https://github.com/networked-aframe/networked-aframe/commits/master/examples/js/forward.component.js

Reply:
added